package.json

```diff
@@ -8,10 +8,11 @@
     "prod": "pnpm run migrate && tsx --env-file-if-exists=.env --dns-result-order=ipv6first bin/server.ts",
     "dev": "pnpm run migrate && tsx watch --env-file-if-exists=.env --dns-result-order=ipv6first bin/server.ts",
     "list:routes": "tsx --env-file-if-exists=.env bin/routes.ts",
-    "test": "pnpm run migrate:test && tsx --env-file-if-exists=.env.test --test",
-    "test:ci": "tsx --test --test-reporter=@reporters/github --test-reporter-destination=stdout --test-reporter=spec --test-reporter-destination=stdout",
+    "test": "pnpm run migrate:test && tsx --env-file-if-exists=.env.test --test --test-concurrency=1",
+    "test:ci": "tsx --test --test-reporter=@reporters/github --test-reporter-destination=stdout --test-reporter=spec --test-reporter-destination=stdout --test-concurrency=1",
     "check": "tsc && biome check .",
     "check:ci": "tsc && biome ci --reporter=github .",
+    "check:coverage": "c8 --check-coverage --lines 100 pnpm test -- --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=coverage/tmp/lcov.info",
```

Comment:
For now, this is just to give an idea of where we stand, but we may want to make it part of CI to check code coverage once we get it high enough.

Could probably also already include it as like a PR comment or something.